### ISSUE TITLE
Is it possible to import psk with textures ??

### ISSUE THREAD

**Issue by ag2s20150909** [Befzz/blender3d_import_psk_psa#57]

**Comment by Blenux** [comment]
Yes would be good but, you would only get basic texture setup at the most, like Diffuse or Normal textures, packed textures can differ from game to game and would be nightmare to add.

**Comment by SolarisBravo** [comment]
No, but the .psk format isn't capable of complex materials - in order to get something that looks half-decent you'd have to redo it from scratch anyway.

Materials really aren't too hard, it's just a matter of plugging in 2-3 textures.

**Comment by Tosyk** [comment]
it would be enough for me at least to have all these textures in the proper material loaded
![image](https://user-images.githubusercontent.com/24209966/125753792-24ae93ce-8c92-4841-9b57-686323e6822b.png)
known textures (diffuse and normal) can be connected to shader and other can be floating around so I don't need to do same actions over and over again.

here's examples of structure the umodel can provided:
[ATATPilot.zip](https://github.com/Befzz/blender3d_import_psk_psa/files/6821469/ATATPilot.zip)
[WallrunLizard.zip](https://github.com/Befzz/blender3d_import_psk_psa/files/6821478/WallrunLizard.zip)

**Comment by tc490225** [comment]
got basic diffuse and normal textures working..  im a github noob so please forgive my ignorance on where\how to get the file out.
[https://github.com/tc490225/blender3d_import_psk_psa](url)
